models/user.rb

```diff
@@ -61,12 +61,17 @@ def to_hash(params={})
           ).count
 
           # Note that the comments may have been responses to a thread not started by author_id.
-          comment_thread_ids = Comment.where(
+
+          # We need to include the comment_thread in thie query to prevent an N+1 query
+          # when we iterate through all of the comments to get their thread context
+          comment_thread_ids = Comment.includes(:comment_thread).where(
```

Comment:
@dan-f -- I commented about this above, but Github shows it as an outdated diff now.

https://github.com/edx/cs_comments_service/blob/bderusha/forum-filter-standalone-threads-TNL-2971/models/user.rb#L67